tasks/actorstate/reward_test.go

```diff
@@ -41,7 +43,7 @@ func TestRewardExtractV0(t *testing.T) {
 	stateCid, err := mapi.Store().Put(ctx, state)
 	require.NoError(t, err)
 
-	minerAddr := tutils.NewIDAddr(t, 00)
+	minerAddr := tutils.NewIDAddr(t, 0o0)
```

Comment:
May be nothing, but curious why this was needed?

Reply:
Not my doing. I think gofumpt does it. Numbers starting with 0 are octal in Go, and they added `0o` as a consistent way to write them to match `0x`